lib/Elastica/Multi/ResultSet.php

```diff
@@ -39,44 +37,13 @@ class ResultSet implements \Iterator, \ArrayAccess, \Countable
     /**
      * Constructs ResultSet object.
      *
-     * @param \Elastica\Response       $response
-     * @param array|\Elastica\Search[] $searches
+     * @param \Elastica\Response $response
+     * @param BaseResultSet[]
```

Comment:
Not sure how phpdoc is going to work with this, but I would assume the type should still be called ResultSet?